modules/cnv_vm_console_web.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-vm-console-web]]
+== Connecting to a virtual machine console
+
+You can connect to the VNC and serial consoles of a running virtual machine from the *Virtual Machine Details* screen of the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines*  from the side menu.
+. Select a Virtual Machine.
+. Click *Consoles*. The VNC console opens by default.
+. Use the *Console* drop-down list to change the console type. Unless the *Disconnect before switching* checkbox is active, you must click *Connect* to open a new console.
```

Comment:
you don't need to click "connect". The "disconnect before switching" means that when you switch between consoles, the previous always disconnects. E.g.:
1: the "disconnect before switching" is not checked
2: Im on the VNC console now
3: in the listbox switch to serial
4: the VNC console connection stays open and an additional connection will be opened to the serial console. You will be presented in the UI only the serial but on the background the connection to the VNC will be preserved
5: if you now switch back to the VNC, it will be very fast because no reconnect is needed. The disadvantage is that you have more opened connections.

If you have the checkbox checked, the consoles will be connected and disconnected every time you switch. As a result you will have less connections opened but every switch will be slower. But from user PoV not a big difference.